src/lib/utils.js

```diff
@@ -0,0 +1,18 @@
+import { debounce, throttle } from 'lodash';
+
+export const isFunction = fn => typeof fn === 'function';
+
+export const listHandle = {
+  debounce,
+  throttle,
+};
+
+export const getHandle = (type) => {
```

Comment:
We can simplify this to
```
export const getHandle = type => listHandle[type];
```

Anyway, it returns either function or `undefined`.

Reply:
I handled in case this return `undefined` value